test/FastTests/Server/Documents/Indexing/BasicAnalyzers.cs

```diff
@@ -238,8 +238,8 @@ public override (ICollection<string> Static, ICollection<string> Dynamic) GetEnt
             throw new NotImplementedException();
```

Comment:
What did you change in the `src/Raven.Server/ServerWide/OperationCancelToken.cs ` file?

Reply:
breaking line has been removed